core/src/main/java/io/grpc/LoadBalancer2.java

```diff
@@ -317,6 +322,77 @@ public String toString() {
   }
 
   /**
+   * Provides arguments for a {@link SubchannelPicker#pickSubchannel(PickSubchannelArgs)}.
+   */
+  public abstract static class PickSubchannelArgs {
```

Comment:
Since LB impls will provide this object, we should make it convenient to create, probably a final class with a builder.